### ISSUE TITLE
Point should be able to parse more than two arguments

### ISSUE THREAD

**Issue by raphaelschaad** [caseycesari/GeoJSON.js#31]
This is needed to parse a third altitude argument, but could be used for even more. From the GeoJSON Sepc:

> A position is represented by an array of numbers. There must be at least two elements, and may be more. The order of elements must follow x, y, z order (easting, northing, altitude for coordinates in a projected coordinate reference system, or longitude, latitude, altitude for coordinates in a geographic coordinate reference system). Any number of additional elements are allowed -- interpretation and meaning of additional elements is beyond the scope of this specification.
http://geojson.org/geojson-spec.html#positions

**Comment by kdavidmoore** [comment]
Hi there, I will be submitting a pull request for this shortly (added support for up to 3 arguments).

FYI the latest RFC caps the number of arguments at 3:
```
Implementations SHOULD NOT extend positions beyond three elements
   because the semantics of extra elements are unspecified and
   ambiguous.  Historically, some implementations have used a fourth
   element to carry a linear referencing measure (sometimes denoted as
   "M") or a numerical timestamp, but in most situations a parser will
   not be able to properly interpret these values.  The interpretation
   and meaning of additional elements is beyond the scope of this
   specification, and additional elements MAY be ignored by parsers.
```
https://tools.ietf.org/html/rfc7946#section-3.1.1

**Comment by caseycesari** [comment]
Fixed in #37

**Comment by raphaelschaad** [comment]
🙌
